README.md

```diff
@@ -1,23 +1,49 @@
-# Overview
-This repo leverages a semi-mechanistic renewal approach to jointly fit COVID-19 hospital admissions data in the US and viral concentrations in wastewater to forecast hospital admissions.
-This codebase was used to produce forecasts for the 2023-2024 season, both in real-time and retrospectively.
-We are currently developing an R package called [wwinference](https://github.com/CDCgov/ww-inference-model) that uses the model developed here and wraps it in a user-friendly interface.
-We plan on using this package to produce regular forecasts and evaluate retrospective forecast performance.
-To try out running this model on your own data, we encourage you to check out that R package and codebase.
+# Bayesian generative modeling for heterogeneous wastewater data applied to COVID-19 forecasting
+This repository contains the code to generate the results of evaluating retrospectively the forecast performance of a wastewater-informed forecasting model, both compared to a model without wastewater data and compared to other models submitted to the  [COVID-19 Forecast Hub](https://github.com/reichlab/covid19-forecast-hub/tree/master) over the 2023-24 epidemic season.
+The model is run using the [`wwinference` R package](https://github.com/CDCgov/ww-inference-model), please see that GitHub repository for a [mathematical description](https://github.com/CDCgov/ww-inference-model/blob/main/model_definition.md) of the model and details on how to install the package and run the model.
```

Comment:
Could link to the docs first and then the Github repo second if you wanted. Very optional.

```suggestion
The model is run using the [`wwinference` R package](https://cdcgov.github.io/ww-inference-model). Please see that package's [GitHub repository](https://github.com/cdcgov/ww-inference-model/) for a [mathematical description](https://github.com/CDCgov/ww-inference-model/blob/main/model_definition.md) of the model and details on how to install the package and run the model.
```